README.md

```diff
@@ -31,11 +31,52 @@ let logger = Logger(label: "example") { _ in
 }
 ```
 
-### Format
-There are a number of ways of customizing the format.
+*Example:*
 
-#### BasicFormatter
-`BasicFormatter` allows you to set the sequence of `LogComponents`, a separator and automatically processes them for each new log message.
+```swift
+let logger = Logger(label: "example") { _ in 
+	return LoggingFormatAndPipe.Handler(
+		formatter: BasicFormatter.adorkable,
+		pipe: LoggerTextOutputStreamPipe.standardOutput
+	)
+}
+```
```

Comment:
This is weird; I didn't see any extension that would allow this to compile in this repo... did I miss something?

In general this isn't reallythe encouraged way to use the swift-log APIs. the way is to use the bootstrap:

```swift
import Logging

LoggingSystem.bootstrap { _ in 
LoggingFormatAndPipe.Handler(
 		formatter: BasicFormatter.adorkable,
 		pipe: LoggerTextOutputStreamPipe.standardOutput
 	)
}
```

Would you be ok with recommending that style? We're trying to have the same style of initializing logging systems exposed by all logging backends; WDYT?